app/mailers/planning_application_mailer.rb

```diff
@@ -115,4 +115,16 @@ def validation_request_closure_mail(planning_application)
       reply_to_id: @planning_application.local_authority.reply_to_notify_id
     )
   end
+
+  def neighbour_consultation_letter_copy_mail(planning_application, email)
+    @planning_application = planning_application
+    @consultation = planning_application.consultation
+
+    view_mail(
```

Comment:
what is `view_mail`? not saying it's wrong, I just don't know what it is!

Reply:
It comes from https://github.com/dxw/mail-notify